rustls-libssl/dist/test/deb-ubuntu-22/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y curl nginx
+ADD rustls-libssl_*.deb /root
+ADD --chown=nobody:nogroup server.cert /
+ADD --chown=nobody:nogroup server.key /
+ADD ca.cert /root
+ADD nginx.conf /etc/nginx/sites-enabled/default
+RUN dpkg -i /root/rustls-libssl_*.deb
+
+# nb, this only affects systemd runs, but check it works at least
+RUN rustls-libssl-nginx enable
+
+CMD /usr/bin/with-rustls-libssl nginx -g 'env LD_LIBRARY_PATH; daemon off;'
```

Comment:
What does the `env` directive end up doing here? The [page I found](https://nginx.org/en/docs/http/ngx_http_core_module.html) listed from the docs for `-g` doesn't mention `env` :thinking:

Reply:
I don't actually think this is required, so I'll remove it -- I guess nginx forks its workers so any environment that affects the dynamic linker don't apply a second time for forked processes. The docs are in https://nginx.org/en/docs/ngx_core_module.html#env

Reply:
Ah ok, passing through some parts of the env explicitly to a forked worker process makes sense :+1: